site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+    //
+    // +optional
+    // +listType=map
+    // +listMapKey=type
+    // +kubebuilder:validation:MaxItems=8
+    Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+```
+
+### Hierarchy
+Each policy MAY include default or override values. Default values are given
+precedence from the bottom up, while override values are top down. That means
+that a default attached to a Backend will have the highest precedence among
+default values while an override value attached to a GatewayClass will have the
+highest precedence overall.
+
+![Ingress and Sidecar Hierarchy](/geps/images/713-hierarchy.png)
+
+To illustrate this, consider 3 resources with the following hierarchy:
+A > B > C. When attaching the concept of defaults and overrides to that, the
+hierarchy would be expanded to this:
+
+A override > B override > C override > C default > B default > A default.
+
+Note that the hierarchy is reversed for defaults. The rationale here is that
+overrides usually need to be enforced top down while defaults should apply to
+the lowest resource first. For example, if an admin needs to attach required
+policy, they can attach it as an override to a Gateway. That would have
+precedence over Routes and Services below it. On the other hand, an app owner
+may want to set a default timeout for their Service. That would have precedence
+over defaults attached at higher levels such as Route or Gateway.
+
+If using defaults and overrides, each policy resource MUST include 2 structs
+within the spec. One with override values and the other with default values.
+
+In the following example, the policy attached to the Gateway requires cdn to
+be enabled and provides some default configuration for that. The policy attached
+to the Route changes the value for one of those fields (includeQueryString).
+
+```yaml
+kind: GKEServicePolicy # Example of implementation specific policy name
+spec:
+  override:
+    cdn:
+      enabled: true
+  default:
+    cdn:
+      cachePolicy:
+        includeHost: true
+        includeProtocol: true
+        includeQueryString: true
+  targetRef:
+    kind: Gateway
+    name: example
+---
+kind: GKEServicePolicy
+spec:
+  default:
+    cdn:
+      cachePolicy:
+        includeQueryString: false
+  targetRef:
+    kind: HTTPRoute
+    name: example
+```
+
+In this final example, we can see how the override attached to the Gateway has
+precedence over the default drainTimeout value attached to the Route. At the
+same time, we can see that the default connectionTimeout attached to the Route
+has precedence over the default attached to the Gateway.
+
+![Hierarchical Policy Example](/geps/images/713-policy-hierarchy.png)
+
+#### Supported Resources
+It is important to note that not every implementation will be able to support
+policy attachment to each resource described in the hierarchy above. When that
+is the case, implementations MUST clearly document which resources a policy may
+be attached to.
```

Comment:
We should document the conformance level and then mention this paragraph.

Reply:
I see that you have done this at the bottom of the doc, combining these makes sense to me.